internal/pkg/vaultsecrets/validation.go

```diff
@@ -30,7 +30,7 @@ func RequireVaultSecretsAppName(ctx *cmd.Context, appName string) error {
 	%v
 	`,
 		cs.String("Vault Secrets application name must be configured before running the command.").Color(cs.Orange()),
-		cs.String("$ hcp profile set vault-secrets/app_name <app_name>").Bold(),
+		cs.String("$ hcp profile set vault-secrets/app <app_name>").Bold(),
```

Comment:
I think this error output should also indicate that the `--app` flag can be used as well.

Reply:
Added 
```
> make go/build && ./bin/hcp vs secrets create 
Vault Secrets application name must be configured before running the command.

To set the Vault Secrets application name interactively, run:

$ hcp profile set vault-secrets/app <app_name>

Alternatively, you can set the Vault Secrets applicaton name on the active pofile using the command:

$ hcp profile init --vault-secrets

If you prefer specifying the Vault Secrets application name directly via the command line, use:

$ hcp vault-secrets secrets --app <app_name> <sub-cmd>
```

Reply:
This is just my opinion, but I think that is getting a bit long. Maybe something like:

```
Vault Secrets application name must be set.

Set the app name using the --app flag or set the app name on your active profile one of the following commands:

  $ hcp profile set vault-secrets/app <app_name>
  $ hcp profile init --vault-secrets
```

Reply:
https://github.com/hashicorp/hcp/pull/75